src/components/tutorialComponent/index.tsx

```diff
@@ -92,7 +92,12 @@ const TutorialDisplay: React.FC = (): JSX.Element => {
     return tutorial.tutorial_title === tutorialName;
   });
 
-  const tutorialData = data.instructions[tutorialStep - 1].content.map(
+  const tutName: string = localStorage.getItem("tutName") || data.tutorial_name;
+
+  const tutStepString: string = localStorage.getItem(tutName) || "1";
+  const tutStep: number = parseInt(tutStepString, 10);
+
+  const tutorialData = data.instructions[tutStep - 1].content.map(
     (element: Content, index: number) => {
       if (element.type === "text")
         return <p key={tutorialName + index.toString()}>{element.value}</p>;
```

Comment:
Since you're now using local storage for the tutorial name, it might be a good idea to phase out any bits of global state that aren't actually applicable so we don't have redundant code.